languages/elixir/exercises/concept/access/.docs/introduction.md

```diff
@@ -0,0 +1,17 @@
+Elixir uses code **Behaviours** to provide a common generic interface for different data while facilitating specific implementations for each type of data. One such example is the **Access Behavior**
+
+## Access Behaviour
+
+The _Access Behaviour_ provides a common interface for retrieving key-based data from a data structure: like from a key-value map structure. Elixir supports the _Access Behaviour_ for maps and keyword lists, but let's look at its use for maps to get a feel for it. _Access Behavior_ specifies that when you have a map, you may follow it with _square brackets_ and then use the key to retrieve the value associated with that key.
```

Comment:
I'm not sure which form you intended to use, but "Access Behavior" is mixed up with "Access Behaviour", and I guess we should stick to just one.